Tests/UnrarTests/EntryTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+import Cunrar
+@testable import Unrar
+
+final class EntryTests: XCTestCase {
+
+    func testCanCreateEntry_withDateHeaderZero() throws {
+        // Given
+        var header = RARHeaderDataEx()
+        header.MtimeLow = 0
+        header.MtimeHigh = 0
+
+        // When
+        let entry = Entry(header)
+
+        // Then
+        XCTAssertEqual(entry.fileName, "")
+    }
+
```

Comment:
```suggestion
      func testCanCreateEntry_withDateHeader() {
        var header = RARHeaderDataEx()
        header.MtimeLow = 848753920
        header.MtimeHigh = 29986355
        let entry = Entry(header)
        XCTAssertEqual(entry.modified, Date(timeIntervalSince1970: 1234567890))
    }
```

Reply:
Thanks for this! I wanted to write that test but I wasn't sure what values to use, because I saw you bitshift and OR those, and I wasn't sure how to reverse that operation 😅